src/main/java/com/hubspot/jinjava/el/ext/ExtendedParser.java

```diff
@@ -427,53 +427,30 @@ protected AstRangeBracket createAstRangeBracket(
   }
 
   @Override
-  protected AstNode value() throws ScanException, ParseException {
```

Comment:
This diff is really weird, but basically I moved the parsing of our custom operators from the `default` of `value` to the `default` of `mul`.

Reply:
With my recent changes, the diff makes more sense now.